user-doc.md

```diff
@@ -0,0 +1,106 @@
+- Display Precipitation Baseline for Feb.
+
+These selectors determine what is displayed on the map.
+
+_Variable_: To select the observation variable, click on one of the three buttons labelled **Precipitation**, **T<sub>min</sub>**, or **T<sub>max</sub>**. The selected button is highlighted.
+
+_Value_: To select the value shown, click on one of the three buttons labelled **Anomaly**, **Monthly**, or **Baseline**.
+
+_Month_: To select the period month, click or drag the top slider to the desired month. 
+
+- To decrement or increment the month selected, click the **-*N*** or **+*N*** button below the month slider. (_N_ here is the number of months by which to change the selection. The default is 1 month, so the buttons are labelled **-1** and **+1**.) 
+- To change the increment, use the drop-down between the buttons. 
+- These buttons can be useful to step uniformly through the months available while looking at the map. 
+- Note that decrementing or incrementing the month carries across year boundaries. For example, incrementing +1 from Dec 2020 changes the selection to Jan 2021.
+
+_Year_: To select the period year, click or drag the top slider to the desired year.
+
+- To decrement or increment the year selected, click the **-*N*** or **+*N*** button below the year slider. (_N_ here is the number of years by which to change the selection. The default is 1 year, so the buttons are labelled **-1** and **+1**.)
+- To change the increment, use the drop-down between the buttons. 
+- These buttons can be useful to step uniformly through the years available while looking at the map.
+
+## Map display
+
+The map display consists of several parts. From top to bottom, roughly:
+
+- _Title_: Indicates the variable, value, and units displayed by the coloured markers.
+- _Colour scale_: Indicates the value ranges represented by the marker colours on the map.
+- _Base map_: A low-colour, unobtrusive map of the region covered by the database, designed to allow marker colours to be read easily.
+- _Zoom in/out controls_: (Upper left of map) These are also controlled by the mouse scroll wheel.
+- _Marker visibility control_: (Upper right of map) Selects which of the three types of markers to display on the map: anomaly, monthly station, and/or baseline station. See below for details.
+- _Markers_: A coloured circle or black dot centred on the location of each station with data, representing the data at that station. Click on a marker to see station and data details. See below for details.
+
+### Map markers 
+
+The central feature of the map is of course the markers, which represent data at each station.
+
+There are three categories of markers:
+- _Baseline markers_: A small black dot at the location of each station for which there is baseline data for the selected variable and month.
+- _Monthly data markers_: A small black dot at the location of each station for which there is monthly data for the selected variable and month.
```

Comment:
what's the difference in appearance (if any) between these two black dot markers? If i understand correctly that these are not visible by default, maybe say so?